app/views/posts/show.html.erb

```diff
@@ -20,7 +20,7 @@
     </div>
     <%= render '/comments/new' %>
     <div class="comment-section">
-        <% @post.most_recent_comments.each do |c|%>
+        <% @post.most_recent_comments.includes([:user]).each do |c|%>
```

Comment:
- [ ] Good implementation of addressing N+1 problem 👍🏻

Reply:
Thank you so much